data/korath/korath variants.txt

```diff
@@ -581,8 +581,8 @@ ship "Ra'gru Ik 618" "Ra'gru Ik 618 (Crippled)"
 	gun "Fire-Lance"
 	gun "Fire-Lance"
 	gun
```

Comment:
```suggestion
	gun "Fire-Lance"
	plyon
	gun "Fire-Lance"
	pylon
	gun
	pylon
```

Reply:
Just a minor tweak request: I have been following the pattern established by the guns and turrets of having all the guns first, then all the pylons, then the turrets, keeping them grouped together. Could I get you to modify your suggestions to follow this pattern too please?